todo_app/browser_tests/chrome_test.py

```diff
@@ -0,0 +1,53 @@
+import os, requests, json
+import pytest
+import todo_app.app
+from selenium import webdriver
+from todo_app.trello_api import create_trello_board, delete_trello_board
+from threading import Thread
+import dotenv
+
+
+@pytest.fixture(scope='module')
+def test_app():
+    
+    # Create the new board & update the board id environment variable
+    test_board_id = create_trello_board()
+    os.environ['BOARD_ID'] = test_board_id
```

Comment:
Neither of the browser tests seems to load the dotenv file; I'm not sure how you're running locally but I'd expect this stage to load the `.env` file (similarly to your integration test); for me this caused errors due to missing values

Reply:
OK, I worked this out and added load_dotenv() to my config file which seems to have fixed the issue.
I wasn't running the tests from the command line, I was running from within vscode, so it was skewing my results